test/es-module/test-esm-tla-unfinished.mjs

```diff
@@ -105,16 +107,29 @@ describe('ESM: unsettled and rejected promises', { concurrency: !process.env.TES
   });
 
   it('should exit for an unsettled TLA promise and respect explicit exit code via stdin', async () => {
-    const { code, stderr, stdout } = await spawnPromisified(execPath, [
+    const { code, stdout, stderr } = await spawnPromisified(execPath, [
       '--no-warnings',
       fixtures.path('es-modules/tla/unresolved-withexitcode.mjs'),
     ]);
 
+    assert.strictEqual(stdout, 'the exit listener received code: 42\n');
```

Comment:
Same thing here, the order is not ideal. Consider using `deepStrictEqual` since we’re not using assert.match

Reply:
actually I just noticed that this test is the exact same as the one below, I might just as well get rid of it